### PR TITLE
chore: improve auth and device error messages for the troubleshoot commands

### PR DESCRIPTION
**Pull request by tranchitella**
Ticket: MEN-5428
Changelog: title

Signed-off-by: Fabio Tranchitella <fabio.tranchitella@northern.tech>

### CODE REVIEW

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
@mender-test-bot start pipeline

**Comment by tranchitella** [comment]
Failure in `tests/test_mender_gateway.py::TestMenderGatewayOpenSource::test_deployment_two_devices_parallel_updates` is unrelated.

File: cmd/root.go [hunk 1]
```diff
@@ -72,7 +72,9 @@ var rootCmd = &cobra.Command{
 // Execute adds all child commands to the root command and sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute() {
-	err := rootCmd.LocalFlags().Parse(os.Args)
+	lflags := rootCmd.LocalFlags()
+	lflags.ParseErrorsWhitelist.UnknownFlags = true
+	err := lflags.Parse(os.Args)
```
Comment (oleorhagen) [inline cmd/root.go:77]: Q:
Isn't this an error message we actually want?

What happens if I misspell a flag?

File: cmd/portforward.go [hunk 2]
```diff
@@ -198,14 +194,28 @@ func (c *PortForwardCmd) Run() error {
 	ctx, cancelContext := context.WithCancel(context.Background())
 	defer cancelContext()
 
+	tokenPath, err := getDefaultAuthTokenPath()
+	if err != nil {
+		return errors.Wrap(err, "Unable to determine the auth token path")
```
Comment (oleorhagen) [inline cmd/portforward.go:199]: S:
`Unable to determine the _default_ auth token path` ?

**Review by oleorhagen — APPROVED** [review]
Looks fine to me

File: cmd/root.go (hunk 1 above)
Comment (tranchitella) [reply cmd/root.go:77]: This is the global parser, bound to the root command. The different commands will still report wrong flags:

```
mender-cli (men-5428) $ ./mender-cli login --blah
Configuration file not found. Continuing.
Error: unknown flag: --blah
Usage:
  mender-cli login [flags]
```

**Comment by oleorhagen** [comment]
@mender-test-bot cherry-pick to: `1.7.x`

**Pull request merged**